kernel/bpf/helpers.c

```diff
@@ -2939,6 +2939,28 @@ __bpf_kfunc void bpf_iter_bits_destroy(struct bpf_iter_bits *it)
 	bpf_mem_free(&bpf_global_ma, kit->bits);
 }
 
+/**
+ * bpf_copy_from_user_str() - Copy a string from an unsafe user address
+ * @dst:             Destination address, in kernel space.  This buffer must be at
+ *	                 least @dst__szk bytes long.
+ * @dst__szk:        Maximum number of bytes to copy, including the trailing NUL.
+ * @unsafe_ptr__ign: Source address, in user space.
+ *
+ * Copies a NUL-terminated string from userspace to BPF space.
+ */
+__bpf_kfunc int bpf_copy_from_user_str(void *dst, u32 dst__szk, const void __user *unsafe_ptr__ign)
+{
+	if (unlikely(!dst__szk))
+		return 0;
+
+	int ret = strncpy_from_user(dst, unsafe_ptr__ign, dst__szk);
+
```

Comment:
nit: no need for empty line

Reply:
But 'scripts/checkpatch.pl' told me to do it.

Reply:
because you need to declare `int ret;` at the top of the function in one variable declaration block